docs/rafter/10-02-rafter-troubleshooting.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Upload Service returns 502 code - Bad Gateway
+type: Troubleshooting
+---
+
+It may happen that the Upload Service returns 502 Bad Gateway with `The specified bucket does not exist` message. It means that the bucket has been removed or renamed.
```

Comment:
```suggestion
It may happen that the Upload Service returns `502 Bad Gateway` with the `The specified bucket does not exist` message. It means that the bucket has been removed or renamed.
```